pages/linux/locate.md

```diff
@@ -9,7 +9,7 @@
 
 - Look for a file by its exact filename (a pattern containing no globbing characters is interpreted as `*pattern*`):
 
-`locate */{{filename}}`
+`locate */{{path/to/file}}`
```

Comment:
```suggestion
`locate */{{filename}}`
```

Reply:
[Invalid](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) format suggested. Conversation was marked as resolved: incorrect suggestion.

Reply:
Changes will be applied: my comment was wrong. Unresolving....